testing/helpers/fileManagerHelpers.js

```diff
@@ -129,6 +130,10 @@ export class FileManagerWrapper {
         return $(`.${Consts.DROPDOWN_MENU_LIST_CLASS} .${Consts.DROPDOWN_MENU_CONTENT_CLASS} .${Consts.DROPDOWN_MENU_LIST_ITEM_CLASS}`)[childIndex];
     }
 
+    getItemViewPopupListItem(childIndex) {
```

Comment:
We haven't any popups in `itemView `widget. This method applies to the toolbar view switcher, so it should be reflect it in it's name. And you should use `'dx-filemanager-toolbar-viewmode-item'` class name in order to increase selector specificity.